packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -335,7 +335,7 @@ class RawGestureDetectorState extends State<RawGestureDetector> {
   /// the gesture detector should be enabled.
   void replaceGestureRecognizers(Map<Type, GestureRecognizerFactory> gestures) {
     assert(() {
-      if (!RenderObject.debugDoingLayout) {
+      if (!(context as Element).owner.debugDoingLayout) {
```

Comment:
This is the one case where the owner field on Element is used. If I'm backing out those changes I'll need to do something else here. Can I temporarily disable this assertion?